commands/cli/README.md

```diff
@@ -23,3 +23,51 @@ Options:
   --version   Show version number                                      [boolean]
   -h, --help  Show help                                                [boolean]
 ```
+
+## How to run nebula cli locally and globally
+
+Requirements:
+
+- Node.js
+- yarn
+
+Clone the application:
+
+```sh
+git clone https://github.com/qlik-oss/nebula.js
+```
+
+From the root directory, you can run the following command to install all the necessary dependencies of nebula cli:
```

Comment:
```suggestion
From the root directory, run the following command to install all the necessary dependencies of nebula cli:
```
Better flow if you omitt 'you can', same thing below.

Reply:
Sure